### PR TITLE
#164797111 User should be able to see their profile

### PR DESCRIPTION
**Pull request by NedyUdombat**
#### What does this PR do?
Gives the user access to his/her profile

#### Description of Task to be completed?
- [x] Implement user profile page

#### What are the relevant pivotal tracker stories?(if applicable)
#164797111

#### Screenshots
loader
Screenshot 2019-05-09 at 7 13 30 PM

Screenshot 2019-05-07 at 7 42 30 AM

### CODE REVIEW

File: src/components/presentationals/ArticleLists/ArticleLists.test.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    id: 123456789,
+  },
+];
+const emptyArticles = [];
+
+describe('ArticleLists', () => {
+  it('should render correctly', () => {
+    const component = mount(<ArticleLists articles={articles} />);
+
```
Comment (Japhethca) [inline src/components/presentationals/ArticleLists/ArticleLists.test.jsx]: We can remove this line space since this is just a two line test.

Comment (NedyUdombat) [reply src/components/presentationals/ArticleLists/ArticleLists.test.jsx]: This is has been removed

File: src/components/presentationals/ProfileSidebar/ProfileSidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      </div>
+      <div style={buttonStyles}>
+        <Button className="btn-white" value="FOLLOW" type="submit" />
+      </div>
+      <hr className="sidebar-hr" />
+    </div>
+  );
+};
+
+const buttonStyles = {
```
Comment (Japhethca) [inline src/components/presentationals/ProfileSidebar/ProfileSidebar.jsx]: Let's use className to add this style.

Comment (NedyUdombat) [reply src/components/presentationals/ProfileSidebar/ProfileSidebar.jsx]: This has been resolved @Japhethca

Comment (Japhethca) [reply src/components/presentationals/ProfileSidebar/ProfileSidebar.jsx]: But this `buttonStyles` is still here.

**Review by Japhethca — APPROVED** [review]

**Pull request merged**